packages/modules/src/SessionKeyManagerModule.ts

```diff
@@ -142,6 +142,15 @@ export class SessionKeyManagerModule extends BaseValidationModule {
       throw new Error('sessionID or sessionValidationModule should be provided.')
     }
 
+    const leafDataHex = ethers.utils.keccak256(
```

Comment:
I think it should be like below,

```
const leafData = hexConcat([
        hexZeroPad(ethers.utils.hexlify(validUntil), 6),
        hexZeroPad(ethers.utils.hexlify(validAfter), 6),
        hexZeroPad(mockSessionValidationModule.address, 20),
        sessionKeyData,
      ]);
```

Reply:
```
const leafDataHex = hexConcat([
      hexZeroPad(ethers.utils.hexlify(sessionSignerData.validUntil), 6),
      hexZeroPad(ethers.utils.hexlify(sessionSignerData.validAfter), 6),
      hexZeroPad(sessionSignerData.sessionValidationModule, 20),
      sessionSignerData.sessionKeyData
    ])

    // Generate the padded signature with (validUntil,validAfter,sessionVerificationModuleAddress,validationData,merkleProof,signature)
    let paddedSignature = defaultAbiCoder.encode(
      ['uint48', 'uint48', 'address', 'bytes', 'bytes32[]', 'bytes'],
      [
        sessionSignerData.validUntil,
        sessionSignerData.validAfter,
        sessionSignerData.sessionValidationModule,
        sessionSignerData.sessionKeyData,
        this.merkleTree.getHexProof(leafDataHex),
        signature
      ]
    )
```

Reply:
edited above code.

Reply:
fixed in https://github.com/bcnmy/biconomy-client-sdk/pull/264/commits/b3da05f2e9c56973e96d0a7a3bc065aef23f9c18